libs/csm/rest/csm_rest_system_health.py

```diff
@@ -626,3 +626,54 @@ def cluster_operation_signal(
         self.log.info("%s operation on %s POST REST API response : %s",
                       operation, resource, response)
         return True, response
+
+    @RestTestLib.authenticate_and_login
+    @RestTestLib.rest_logout
+    def post_resource_signal(
+            self,
+            req_body: dict,
```

Comment:
Can't we form request body in function itself? As body remains same, only two params we have to pass for it, operation and id.

Reply:
I have considered all the resource type using same post method. So there might be change in body of resource. Hence kept in HA libs.

Reply:
Request body will change as per resource type? I think body will remain same, only param values will change. Please confirm.

Reply:
Did you confirm this?